llvm/lib/Transforms/Vectorize/VPlanUtils.h

```diff
@@ -39,18 +39,37 @@ const SCEV *getSCEVExprForVPValue(VPValue *V, ScalarEvolution &SE);
 
 /// Returns true if \p VPV is uniform after vectorization.
```

Comment:
Independent: documentation should be (more) meaningful. Name should be more accurate.

Reply:
Will check separately